rundeckapp/grails-app/views/common/_sidebar.gsp

```diff
@@ -89,12 +89,13 @@
       </g:link>
     </li>
     <li id="nav-jobs-link" class="${enc(attr: wfselected)}">
-        <g:link controller="menu" action="jobs" class=" toptab ${enc(attr: wfselected)}" params="[project: projectName]">
-            <i class="fas fa-tasks"></i>
-            <p>
-              <g:message code="gui.menu.Workflows"/>
-            </p>
-        </g:link>
+        <g:set var="jobListType" value="${prop.projectPropertyVal([project:projectName,prop:"project.gui.job.list.default"])}"/>
```

Comment:
`jobListType` does not seem to be added to the link params

Reply:
Looks like I forgot to remove that line. It's not needed.